_data/simple-icons.json

```diff
@@ -4015,6 +4015,11 @@
             "hex": "0076A8",
             "source": "https://www.mathworks.com/brand.html"
         },
+        {
+            "title": "Matomo",
+            "hex": "3152A1",
```

Comment:
@Jorys-Paulin Thanks for your work.

The logo itself looks good to me. Just a minor change:

Looking at all the PNG and SVG files from the media kit, as well as at different elements on their website:
The dark blue seems to be `#3152A0` and not  `#3152A1`. Could you change that?

After this it's ready to be merged. 👍🏼

Reply:
Hello, I've updated the color in commit `a8d2465`, linked here: https://github.com/simple-icons/simple-icons/pull/4462/commits/a8d2465f0ec514d7ada2658497a1966dd34de150